src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -46,7 +46,10 @@ public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IF
         public int MaxFileSizeMb { get; set; } = 10;
 
         // Maximum time between successive flushes (seconds)
-        public int FlushFrequencySeconds { get; set; } = 30;
+        public int MaxFlushFrequencySeconds { get; set; } = 30;
+
+        // Start flush frequency low to quickly get Function Execution Events for scaling
+        public int CurrentFlushFrequencySeconds { get; set; } = 1;
```

Comment:
Looks like these are used only inside this type. Can these be marked as `private` ?